README.md

```diff
@@ -16,6 +16,14 @@ npm i svelte-tiptap
 yarn add svelte-tiptap
 ```
 
+If you're still using Svelte 4, you can use version 1 instead:
```

Comment:
Good that's its added, but Instead of giving install instructions, we can point to the v1 tag. https://github.com/sibiraj-s/svelte-tiptap/tree/v1.1.3, since the steps in README is different or will potentially break for v4 users.

Reply:
Can you create a v1 branch instead? This was if I point to it it can still get commits/updates and the link will always point to the latest v1

Reply:
Branches, are not required. We have tags and branch out anytime if needed. Not sure maintain both version until required. Plus, the statement is enough to say this is for latest version.

Also, there are no bugs in v4 so far. we can continue to maintain v5.

Reply:
The reason why I want this is so I can update the README on the v1 branch also, for example the install command should contain the `@1` or else it will install the v2 version which won't work for Svelte 4 users.

Reply:
Okay, I will make a small PR for the v1 tag with the README then